include/fs.h

```diff
@@ -58,7 +58,7 @@ struct fs_mount_t {
 	enum fs_type type;
 	const char *mnt_point;
 	void *fs_data;
-	struct device *storage_dev;
+	void *storage_dev;
```

Comment:
I think it might be helpfully if this will be a pointer to a union. This will make the future code more slef-documenting than the pointer to void. What do you thin?

Reply:
my intention of changing this to void* is, if we need have more info other than device then the application can package that in a struct and pass it to the FS. For example, in NFFS case I thought of sending the no.of Flash areas and device pointer for the specific drive/partition.

Reply:
ok - understood this form the beginning. It might be not good idea to reference variety of sub-types to the union I proposed.
<b>Accepted</b> as is.